src/registry.ts

```diff
@@ -335,6 +335,9 @@ function handleGetEntryErrResponse(err: AxiosError): SignedRegistryEntry {
   // Check if status was 404 "not found" and return null if so.
   if (err.response.status === 404) {
     return { entry: null, signature: null };
+  } else if (err.response.status >= 400) {
```

Comment:
This means that the code following these status code checks are only there to cover `3xx` codes.
I'm just wondering why we can't simply do:
```
if (err.response.status === 404) {
    return { entry: null, signature: null };
}

throw err // rethrow
```
doesn't this accomplish exactly the same. I don't see the value in wrapping the original error the way we do, aside from handling the 404 case.